packages/contracts-ethers/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aragon/core-contracts-ethers",
-  "version": "0.0.1-alpha",
+  "version": "0.1.0-alpha",
```

Comment:
@mathewmeconry Is this correct?

Reply:
@Michael-A-Heuer is it a new feature or just a patch?
I suggest we follow normal semver

Reply:
I think it is because we have two types of `Component` now, one that can be used to send meta transaction and one that doesn't.